TileStache/Core.py

```diff
@@ -144,12 +144,21 @@
 """
 
 import logging
+from sys import modules
 from wsgiref.headers import Headers
-from StringIO import StringIO
-from urlparse import urljoin
+try:
+    from io import BytesIO
```

Comment:
I guess we should be using BytesIO in all cases then. After that we can merge, but let's hold a little on a release. Could you use the master branch on openaddresses for now?